app/src/helpers/utils/manageFocus.ts

```diff
@@ -0,0 +1,26 @@
+export function focusElement(element: HTMLElement): void {
+    if (document.activeElement === element) {
+        return;
+    }
+
+    if (!element.hasAttribute('tabIndex')) {
+        element.setAttribute('tabIndex', '-1');
+        element.addEventListener(
+            'blur',
+            (e) => {
+                e.preventDefault();
+                element.removeAttribute('tabIndex');
+            },
+            { once: true },
+        );
+    }
+
+    element.focus();
+}
+
+export function focusLayoutDiv() {
+    const layoutDiv = document.querySelector('div#layout');
```

Comment:
Would it be better to pass an input ref of the H1 instead ? Generally bad practise to use query Selector in react as document can be undefined

Reply:
Thanks for spotting this!
As it seems quite a big task to keep a global context for h1 ref and populate it on every page, for now I have addressed the problem like this:
 - use `document?.` to prevent any query causing an error
 - add a comment in relevant code to raise attention in case we use SSR in the future (which would break usages of `document` object)